demosplan/DemosPlanCoreBundle/Logic/Statement/DraftStatementService.php

```diff
@@ -195,11 +198,20 @@ public function getDraftStatementList($procedureId, $scope, StatementListUserFil
             } else {
                 $conditions[] = $this->conditionFactory->propertyHasValue($userOrganisationGatewayName, ['organisation', 'gatewayName']);
             }
-
-            if ('own' === $scope || 'ownCitizen' === $scope) {
+            if ('ownCitizen' === $scope) {
+                // ownCitizen bleibt so wie es ist.
                 $conditions[] = $this->conditionFactory->propertyHasValue($user->getId(), ['user']);
                 // add filter to be seen by elasticsearch
-                $filters->setSomeOnesUserId($user->getIdent());
+                $filters->setSomeOnesUserId($user->getId());
+            }
+            if ('own' === $scope) {
+                // own means own orga in this context.
+                // filter out all private draftStatements of own orga if they are mot authored by currentUser
```

Comment:
typo and avoid short term for organisation

```suggestion
                // own means own organisation in this context.
                // filter out all private DraftStatements of own organisation if they are not authored by currentUser
```